pkg/utils/threadsafemultimap.go

```diff
@@ -5,7 +5,10 @@
 
 package utils
 
-import "sync"
+import (
+	"github.com/deckarep/golang-set"
+	"sync"
```

Comment:
Let's order them in 2 groups - built-in Go packages in the first group (at the top) and the 3rd party packages in the second group - separated by a `\n`

Reply:
Done!